src/main/java/io/cryostat/core/agent/Event.java

```diff
@@ -253,4 +253,22 @@ private Element buildMethodElement(Document document) {
 
         return methodElement;
     }
+
+    @Override
+    public String toString() {
```

Comment:
If this is only used here:

https://github.com/cryostatio/cryostat/pull/1126/files#r998580269

then I think this override can actually be removed, and just let the Gson instance within Cryostat deal with this.

Reply:
Removed this since it's no longer necessary.